src/Widget/FieldBuilder.php

```diff
@@ -0,0 +1,1096 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Form\Widget;
+
+use Yiisoft\Arrays\ArrayHelper;
+use Yiisoft\Factory\Exceptions\InvalidConfigException;
+use Yiisoft\Html\Html;
+
+use function array_merge;
+use function is_subclass_of;
+
+class FieldBuilder extends Widget
+{
+    private ?FormBuilder $forms = null;
+    private string $template = "{label}\n{input}\n{hint}\n{error}";
+    private array $divOptions = ['class' => 'form-group'];
+    private array $errorOptions = ['class' => 'help-block'];
+    private array $labelOptions = ['class' => 'control-label'];
+    private array $hintOptions = ['class' => 'hint-block'];
+    private ?bool $enableClientValidation = null;
```

Comment:
I don't think we should port it directly. See https://github.com/yiisoft/validator/issues/43

Reply:
Done.